microshift_updating/microshift-update-options.adoc

```diff
@@ -63,7 +79,12 @@ You can update {op-system-ostree} or {op-system} without updating {microshift-sh
 
 [id="microshift-update-options-simultaneous-microshift-rhel-updates_{context}"]
 == Simultaneous {microshift-short} and operating system updates
-You can update {op-system-ostree} or {op-system} and update {microshift-short} at the same time, on the condition that the versions are compatible. Check for compatibility before beginning an update. First use the {op-system-ostree} and {op-system} documentation specific to your update path to plan and update the operating system. Then use the {microshift-short} update type specific to your update path.
+You can update {op-system-ostree} or {op-system} and update {microshift-short} at the same time, on the condition that the versions are compatible.
+
+* Check for compatibility before beginning an update.
```

Comment:
Maybe number these if the order is important?

Reply:
I feel like I tried that before and got told that made it a procedure. But really this is the order of the workflow steps. Will number them now. TY!